File: UNIDAD_3/Java/Unidad_3_Programa_5.java
 /*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.unidad_3_programa_5;
import java.util.Scanner;
/**
 *
 * @author LaboratorioLTI2
 */
public class Unidad_3_Programa_5 {

    public static void main(String[] args) {
        Scanner sm = new Scanner (System.in);
        double p1, p2, p3, total;
        System.out.println("Escribe precio 1: ");
        p1 = sm.nextDouble ();
          System.out.println("Escribe precio 2: ");
        p2 = sm.nextDouble ();
          System.out.println("Escribe precio 3: ");
        p3 = sm.nextDouble ();
        total = p1 + p2 + p3;
        if (total>=1500){
            total = total - (total * 0.30);
            System.out.println ("El total (30% desc): " + total);
        }else if (total<1500 && total>=1000){
            total = total - (total * 0.20);
            System.out.println ("El total (20% desc): " + total);
        }else if (total<1000 && total>=700){
            total = total - (total * 0.10);
            System.out.println ("El total (10% desc): " + total);
            }else {
            System.out.println ("El total del descuento es de 0%: " + total);
        }
        }
    }

File: UNIDAD_3/C#/Unidad_3_Programa_5.cs
using System;

namespace Unidad_3_Programa_5 // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            double p1, p2, p3, total;
            Console.WriteLine("Escribe precio 1: ");
            p1 = Convert.ToDouble(Console.ReadLine());
            Console.WriteLine("Escribe precio 2: ");
            p2 = Convert.ToDouble(Console.ReadLine());
            Console.WriteLine("Escribe precio 3: ");
            p3 = Convert.ToDouble(Console.ReadLine());
            total = p1 + p2 + p3;
            if (total >= 1500)
            {
                total = total - (total * 0.30);
                Console.WriteLine("El total (30% desc): " + total);
            }
            else if (total < 1500 && total >= 1000)
            {
                total = total - (total * 0.20);
                Console.WriteLine("El total (20% desc): " + total);
            }
            else if (total < 1000 && total >= 700)
            {
                total = total - (total * 0.10);
                Console.WriteLine("El total (10% desc): " + total);
            }
            else
            {
                Console.WriteLine("El total del descuento es de 0%: " + total);

            }
        }
    }
}